cloud-nio/cloud-nio-impl-drs/src/main/scala/cloud/nio/impl/drs/DrsPathResolver.scala

```diff
@@ -241,4 +245,11 @@ object DrsResolverResponseSupport {
         baseMessage + "(empty response)"
     }
   }
+
+  lazy val connectionManager = {
+    val connManager = new PoolingHttpClientConnectionManager()
+    connManager.setMaxTotal(250)
+    connManager.setDefaultMaxPerRoute(250)
```

Comment:
Can you add a brief comment about how you came up with these numbers?

Reply:
Honestly, not really. Its just a shot in the dark. I can explain the max connections per route being the same as max total though.